### ISSUE TITLE
Community bugs

### ISSUE THREAD

**Issue by Frahuenheld** [Fenoxo/sourceTiTS#331]
Some bugs I found on the comments in the homepage. Don't know if they posted this this on the forums yet.

User "xPUREx" got this message trying to flirt to the new queen. He says he choosed assertive option.

Version: 0.5.27
TypeError: Error #1010
at classes::Creature/cockHead()
at classes::Creature/getDescription()
at classes.Parser::ParseEngine/convertDoubleArg()
at classes.Parser::ParseEngine/parseNonIfStatement()
at classes.Parser::ParseEngine/recParser()
at classes.Parser::ParseEngine/recursiveParser()
at classes::TiTS/doParse()
at classes::TiTS/output()
at classes::TiTS/cocklustWivAntsGuv()
at classes::TiTS/buttonClick()

User "Slab Bulkhead" found this while transforming new characters.
1. I went through the naleen nip process to make a herm character with two dicks. She now does a ridiculous amount of lust damage with the ‘Crotch’ option, close to double other options. Is this intended, due to having twice the usual number of cocks, or is it a bug?
2. Are things like muscle tone, body thickness, and the like not fully implemented? I was working on my cowgirl character, and the messages I got from her binge-eating Bovinium made it sound like she was getting ridiculously thick, but the character description still shows her as having a “tight butt” and a “slim midriff”, though she does have “wide thighs”. I saw some discussion about this on the forums, but I wasn’t sure what was in the game or not.
3. Does lactation eventually stop? My cowgirl keeps getting messages about it, but I can’t tell if it does or not, since she’s still listed as lactating and has all the milk-based options.

**Comment by Fenoxo** [comment]
First one: Fixt earlier.

1) Shouldn't be double based on dicks alone, but you might be getting foes that like multiple or large dicks.
2) They should be. Chances are the specific TF is probably more butt/thighs oriented and an oversight, but I'm not gonna try and hunt it down without some quotes from the scene.
3) It does, however quite a few things can make it persist indefinitely - being treated, having the milky perk, etc.

TL;DR: Different issues should go in different reports, and without the original user to provide appropriate clarification and context, most comment bug reports are garbage.
